src/package_manager/package_manager.rs

```diff
@@ -61,6 +64,7 @@ impl PackageManager {
             PackageManager::Off => panic!("no package manager"),
             PackageManager::Deb => "deb".to_string(),
             PackageManager::Rpm => "rpm".to_string(),
+            PackageManager::TreeHub => "thm".to_string(),
```

Comment:
Maybe, that Ostree stuff was just meant for demo, but I think the deprecation should happen in another commit.